src/Grafo/Grafo.cpp

```diff
@@ -46,13 +46,53 @@ Vertice* Grafo::buscarVertice(Tablero* tableroBuscado){
 	this->VerticesDelGrafo->iniciarCursor();
 	Vertice* verticeBuscado = 0;
 	while (!verticeBuscado && this->VerticesDelGrafo->avanzarCursor()){
-		if (this->VerticesDelGrafo->obtenerCursor()->getPeso() == tableroBuscado){
+		if (this->VerticesDelGrafo->obtenerCursor()->getTableroAsociado() == tableroBuscado){
 			verticeBuscado = this->VerticesDelGrafo->obtenerCursor();
 		}
 	}
 	return verticeBuscado;
 }
 
+Arista* Grafo::buscarArista(Vertice* verticeOrigen, Vertice* verticeDestino){
+	Arista* aristaBuscada = 0;
+	bool halladaLaArista = false;
+	verticeOrigen->getAristas()->iniciarCursor();
+	while (! halladaLaArista && verticeOrigen->getAristas()->avanzarCursor()){
+		Arista* aristaActual = verticeOrigen->getAristas()->obtenerCursor();
+		if (aristaActual->getVerticeReceptor() == verticeDestino){
+			aristaBuscada = aristaActual;
+			halladaLaArista = true;
+		}
+	}
+	return aristaBuscada;
+}
+
+int Grafo::buscarRecorridoMinimo(Vertice* verticeOrigen, Vertice* verticeDestino){
+	Cola<Vertice*>* aVisitar = new Cola<Vertice*>;
```

Comment:
aca al principio del método setear todos los `visitado` a `False` con el metodo escrito arriba                     
 `this->setearVerticesANoVisitados()`